test/utils/Assertions.sol

```diff
@@ -109,7 +108,7 @@ abstract contract Assertions is PRBTest, PRBMathAssertions {
     /// @dev Compares two {LockupDynamic.Segment[]} arrays.
     function assertEq(LockupDynamic.Segment[] memory a, LockupDynamic.Segment[] memory b) internal {
         if (keccak256(abi.encode(a)) != keccak256(abi.encode(b))) {
-            emit Log("Error: a == b not satisfied [LockupDynamic.Segment[]]");
+            emit log("Error: a == b not satisfied [LockupDynamic.Segment[]]");
             emit LogNamedArray("   Left", a);
```

Comment:
Should I rename `LogNamedArray` to `log_names_array` and `LogNamedUint128` to `log_named_uint128` to make it look consistent with the [StdAssertions log style](https://github.com/foundry-rs/forge-std/blob/bb4ceea94d6f10eeb5b41dc2391c6c8bf8e734ef/src/StdAssertions.sol#L10C2-L34C54)?

Reply:
Sounds like a good idea